client/src/classes/item.ts

```diff
@@ -12,15 +13,18 @@ export class Item {
   private unit: Unit;
 
   constructor(private name: string, public bought: boolean = false, 
-              public department: Department = null) {
+              public department: Department = null, private template: boolean = false) {
 
     Item.entries++;
```

Comment:
Try to avoid reliance on global state, or in this case state which "reaches out" of its own knowledge context. Sure, it only really reaches out to a static variable context, but still it can lead to problems. Datum should not know or care about where they are.